### ISSUE TITLE
Review AgentSoccer.cs Logic to be consistent with original logic

### ISSUE THREAD

**Issue by Tristan-Dormans** [AlsoKnownAs-Ax/Project2-1-AIML8#55]
Review AgentSoccer.cs Logic to be consistent with original logic to ensure that training the agents without any sensors will 
lead to a similar model to the pretrained one, to review if our AgentSoccer.cs reward logic does not interfere with our base case model. The base case model will be compared to the original pre-trained model to see if we train correctly.
